awscli/customizations/globalargs.py

```diff
@@ -10,6 +10,7 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+import six
```

Comment:
We generally import this from our compat module: `from awscli.compat import six`